contributing/development/configuring_an_ide/visual_studio.rst

```diff
@@ -47,7 +49,7 @@ Project Manager opens a project, the initial process is terminated and the debug
 - Open the **Debugging** section and under **Command Arguments** add two new arguments:
   the ``-e`` flag opens the editor instead of the Project Manager, and the ``--path`` argument
   tells the executable to open the specified project (must be provided as an *absolute* path
-  to the project root, not the ``project.godot`` file).
+  to the project root, not the ``project.godot`` file, also must not contain whitespaces in the path).
```

Comment:
You can pass path with spaces though (not sure about other whitespaces) by using double quotes. Like: `--path "some/path/name with spaces"`.

Reply:
> I could replace it to suggest always using the quotes, which should be the safer option.


Fine to me. But a note in here could also be added:
```suggestion
  to the project root, not the ``project.godot`` file; if the path contains spaces be sure to pass it inside double quotation marks).
```
or something like that.

Reply:
Done